active_storage_base64.gemspec

```diff
@@ -10,9 +10,13 @@ Gem::Specification.new do |s|
   s.authors = ['Nicolas Fabre, Ricky Ricoch']
   s.license = 'MIT'
 
-  #Dependencies
-  s.add_dependency 'rails', '>= 5.2.0'
+  # Dependencies
+  s.add_dependency 'rails', '~> 5.2.0'
 
-  #Development dependencies
-  s.add_development_dependency 'rubocop', '>= 0.49.1'
+  # Development dependencies
+  s.add_development_dependency 'rubocop', '~> 0.56.0 '
+  s.add_development_dependency 'reek', '~> 4.8.1'
+  s.add_development_dependency 'rspec-rails', '~> 3.8.0'
+  s.add_development_dependency 'rails_best_practices', '~> 1.19.0'
```

Comment:
Is `rails_best_practices` useful here? Given that this is a gem and not a Rails app.

Reply:
Since this is an open source project where everyone should be able to contribute to it, I feel that every gem that helps us ensure code quality is welcome. Do you think I should remove it?

Reply:
I think most of its rules won't apply to this, but if some rules do apply, then I don't have a problem with keeping it.